modules/retail/pages/retail-offline.adoc

```diff
@@ -0,0 +1,33 @@
+[[retail.offline]]
+= Offline Use
+
+If the {susemgr} Server is offline, you can still perform a reboot on the terminals.
+If the connection between the branch server and the terminals is lost or unstable, you can still perform an update on the terminals, using caching.
+
+[IMPORTANT]
+====
+Offline features are available only for {susemgr} Server 4.0 and above.
+You must have the Saltboot and PXE formula packages installed.
+====
+
+== Offline Terminal Reboot
+
+If the {susemgr} Server is offline, and a terminal is rebooted, it will fall back to a previously installed image.
+
+This will occur in these situations:
+* If the saltboot.py module on the {susemgr} Server has not been synchronized within a specified time (default value is 60 seconds).
+This is the first step in the usual salt boot procedure, so if it fails, it indicates that there is a problem with the {susemgr} Server.
```

Comment:
This belongs to the first item of this list.  It is the explanation why and not different situation.

Reply:
Yes, it's just a second sentence in the first bullet point. I think I probably want to turn it all into prose eventually, though.